sdk/core/azure-core/test/ut/transport_adapter.cpp

```diff
@@ -143,7 +147,7 @@ namespace Azure { namespace Core { namespace Test {
     auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, host);
 
     // loop sending request
-    for (auto i = 0; i < 500; i++)
+    for (auto i = 0; i < 50; i++)
```

Comment:
Why do we loop 50 times or even 500 times, before?

Reply:
Arbitrary value.  There was an issue in the past when resending request multiple times.

What happens here is that sending a request produces a response from Server which we read from the wire in chunks that are not the same on every call. Repeating the request multiple times expects to be able to face multiple reading scenarios (reading small, medium and big chunks).

Reply:
In that case, why are we changing it?

Reply:
Just trying to save some 1 or 2 seconds while running unit tests.
This is based on that time we talked about loop test taking some seconds to complete on Windows. We can reduce the loop from 500 to 50 and the coverage is still good